server/routes/login.js

```diff
@@ -0,0 +1,27 @@
+const user = require('../models/user.js');
+const express = require('express');
+const router = express.Router();
+const jwt = require('jsonwebtoken');
+const mongoUtil = require('../helpers/mongoUtil.js');
+const ObjectID = require('mongodb').ObjectID;
+
+router.post('/', function(req, res) {
+  mongoUtil.getDb().collection('users').findOne({username: req.body.username}, function(err, user){
```

Comment:
One of the things that needs to be done on the database is to create certain constraints and indexes. A unique index on the username field makes sense here since we don't want two accounts with the same username, plus having an index makes queries faster on this field.